### ISSUE TITLE
Move astropy.samp to pyvo?

### ISSUE THREAD

**Issue by pllim** [astropy/pyvo#155]
Did we discuss this at IVOA Interop? cc @astrofrog

**Comment by astrofrog** [comment]
We didn't really discuss it, but it's certainly an option! (I'd argue that other things like VOTable belong in the core package, but no strong opinions on astropy.samp)

**Comment by pllim** [comment]
I also agree that `votable` should stay in the core since it is so integrated to the unified I/O.

**Comment by funbaker** [comment]
This would also require to de-couple the submodules from the top-module.
(It's not necessary to import the whole `dal` stack for just broadcasting a table.)

**Comment by pllim** [comment]
At Astropy Coordination Meeting 2019, this was decided to be a good idea. But `io.votable` would *stay* in `astropy`.

**Comment by pllim** [comment]
Is there a roadmap for this?

**Comment by tomdonaldson** [comment]
Renewing this effort...

The goal:  We would like to deprecate then remove`astropy.samp` from astropy in favor of `SAMP` capabilities in `pyvo`.

Note that [pyvo/samp](pyvo/samp.py) depends on `astropy.samp`, so that reference will need to change.  This work should ensure that existing `pyvo` SAMP capabilities are retained and that `astropy` SAMP capabilities are supported and documented.

For more details on this see the notes from unmerged PR #239.

**Comment by pllim** [comment]
Please cross link the new PR here when you have one. Thank you so much! 🙏

**Comment by bsipocz** [comment]
I'm removing the milestone from this one as this is not a thing that should block the 1.5 version.
